projects/components/table/src/subcomponents/table-actions.component.html

```diff
@@ -5,21 +5,40 @@
 <mat-menu #menu>
   <ng-template matMenuContent>
     <ng-container *ngFor="let action of actions">
-      <button
-        *ngIf="!action.children; else branch"
-        mat-menu-item
-        [disabled]="action.isDisabledFn && action.isDisabledFn(items)"
-        (click)="action.actionFn(items)"
-      >
-        <mat-icon [style.color]="action.iconColor">{{ action.icon }}</mat-icon>
-        {{ action.label }}
-      </button>
+      <ng-container *ngIf="action.isLoading || !action.hasChildren; else branch">
```

Comment:
the changes here could have a substantial performance impact. all that s ifs and templates are the reason for the current performance problems. with this we have even more of them.